### ISSUE TITLE
Instructions aren't quite clear for me..

### ISSUE THREAD

**Issue by goobertthegoober** [OpenAsar/arrpc#97]
Where should i do _npm install_? I don't know which of them all i use! Is it powershell or the windows console? Node said to do it in other place!

**Comment by nlutterman** [comment]
Either PowerShell or Windows Console (CMD) should work. 

The command itself depends on whether you have added `npm.exe` to your `PATH` variable. If not, you have to refer to the executable's full path when invoking it, like `"C:\Program Files\nodejs\npm" install`

To simplify things, you can add the directory to the `PATH` environment variable like this:   
```dosbatch
setx path "%PATH%;C:\Program Files\nodejs"
```
Then after setting it, re-open whichever command line you're using as they only read it once upon startup.

So in short:

1. Open either PowerShell or CMD
2. Navigate to the base directory containing this repository's code
3. From within that directory:
   - If `PATH` adjusted: `npm install`
   - If `PATH` **NOT** adjusted: `"C:\Program Files\nodejs\npm" install`
4. Then, to start the server:
   - If `PATH` adjusted: `npm run start`
   - If `PATH` **NOT** adjusted: `"C:\Program Files\nodejs\npm" run start`

The server should then be running. That's all that's necessary.

**Comment by goobertthegoober** [comment]
> Either PowerShell or Windows Console (CMD) should work.
> 
> The command itself depends on whether you have added `npm.exe` to your `PATH` variable. If not, you have to refer to the executable's full path when invoking it, like `"C:\Program Files\nodejs\npm" install`
> 
> To simplify things, you can add the directory to the `PATH` environment variable like this:
> 
> ```batchfile
> setx path "%PATH%;C:\Program Files\nodejs"
> ```
> 
> Then after setting it, re-open whichever command line you're using as they only read it once upon startup.
> 
> So in short:
> 
> 1. Open either PowerShell or CMD
> 2. Navigate to the base directory containing this repository's code
> 3. From within that directory:
>    
>    * If `PATH` adjusted: `npm install`
>    * If `PATH` **NOT** adjusted: `"C:\Program Files\nodejs\npm" install`
> 4. Then, to start the server:
>    
>    * If `PATH` adjusted: `npm run start`
>    * If `PATH` **NOT** adjusted: `"C:\Program Files\nodejs\npm" run start`
> 
> The server should then be running. That's all that's necessary.

Uh... Neither `npm install` or `"C:\Program Files\nodejs\npm" install` work. Did i miss something?

**Comment by CanadaHonk** [comment]
The readme was updated to use `npx` instead which should hopefully be easier.
